README.md

```diff
@@ -48,7 +48,7 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AndrewChew
 
 ## Professional Experiences
 
-**Junior Software & Data Engineer, Jarvis Consulting Group (2021-present)**: Develop software using the SDLC process in an agile work environment, delivering high-quality products.
+**Junior Software & Data Engineer, Jarvis Consulting Group (2021-present)**: Develop software using the SDLC process in an agile work environment, delivering high-quality products. Spearheaded daily scrum meetings as team lead for a sprint.
 
 **Produce Clerk Part-Time, Loblaw Companies Limited (2016-present)**: Advise customers on over 400 products leading to improved customer satisfaction. Effectively handle conflict resolution in a sensitive and professional manner ensuring safety in the workplace. Train new employees for day-to-day operations. Successfully completed diversity training.
```

Comment:
**Produce Clerk Part-Time, Loblaw Companies Limited (2016-present)**: Advise customers on over 400 products leading to improved customer satisfaction. Effectively handle conflict resolution in a sensitive and professional manner, ensuring safety in the workplace. Train new employees for day-to-day operations. Successfully completed diversity training.

Reply:
**Events Coordinator, Hope United Church (2012-2016)**: Managed a team of three to organize after-school activities for ten children. Strategized learning sessions for grade school children in Mathematics and Science. Taught children basic piano skills.